src/content/install-dialog.js

```diff
@@ -3,8 +3,8 @@ document.title = details.name + ' - Greasemonkey User Script';
 
 
 function finish() {
-  chrome.windows.getCurrent(null, win => {
-    chrome.windows.remove(win.id);
+  chrome.tabs.getCurrent(curTab => {
+    chrome.tabs.remove(curTab.id);
```

Comment:
1) Why?
2) Desktop vs. Android?

Reply:
Using tabs works on both devices. Android needs to use tabs since it doesn't have windows. On desktop since the installer is opened using a dialog the window is considered a single tab window. When that tab is removed so is the window.